.github/workflows/build_pdf_book.yml

```diff
@@ -0,0 +1,42 @@
+name: Build latest version of PDF Book
+
+on:
```

Comment:
Looks good, can you please explain when exactly this workflow will get triggered.

Reply:
Yes, the workflow will be triggered whenever new commits are performed in examples/tutorials directory in deepchem repository.